src/analysis/processing/qgsalgorithmcellstatistics.cpp

```diff
@@ -0,0 +1,337 @@
+}
+
+QString QgsCellStatisticsAlgorithm::groupId() const
+{
+  return QStringLiteral( "rasteranalysis" );
+}
+
+QString QgsCellStatisticsAlgorithm::shortHelpString() const
+{
+  return QObject::tr( "The Cell statistics algorithm computes a value for each cell of the "
+                      "output raster. At each cell location, "
+                      "the output value is defined as a function of all overlaid cell values of the "
+                      "input rasters.\n\n"
+                      "The output rasters extent and resolution is defined by a reference "
+                      "raster. The following functions can be applied on the input "
+                      "raster cells per output raster cell location:\n"
+                      "<ul> "
+                      "   <li>Sum</li>"
+                      "   <li>Count</li>"
+                      "   <li>Mean</li>"
+                      "   <li>Median</li>"
+                      "   <li>Standard deviation</li>"
+                      "   <li>Variance</li>"
+                      "   <li>Minimum</li>"
+                      "   <li>Maximum</li>"
+                      "   <li>Minority (most frequent value)</li>"
+                      "   <li>Majority (least frequent value)</li>"
+                      "   <li>Range (max-min)</li>"
+                      "   <li>Variety (count of unique values)</li>"
+                      "</ul> "
+                      "Input raster layers that do not match the cell size of the reference raster layer will be "
+                      "resampled using nearest neighbor resampling. The output raster data type will be set to "
+                      "the most complex data type present in the input datasets except when using the functions "
+                      "Mean and Standard deviation (data type is always Float32) or Count and Variety (data type is always Int32).\n"
+                      "<i>Calculation details - general:</i> NoData values in any of the input layers will result in a NoData cell output if the Ignore NoData parameter is not set.\n"
+                      "<i>Calculation details - Count:</i> Count will always result in the number of cells without NoData values at the current cell location.\n"
+                      "<i>Calculation details - Median:</i> If the number of input layers is even, the median will be calculated as the "
+                      "arithmetic mean of the two middle values of the ordered cell input values.\n"
+                      "<i>Calculation details - Minority/Majority:</i> If no unique minority or majority could be found, the result is NoData, except all "
+                      "input cell values are equal." );
+}
+
+QgsCellStatisticsAlgorithm *QgsCellStatisticsAlgorithm::createInstance() const
+{
+  return new QgsCellStatisticsAlgorithm();
+}
+
+void QgsCellStatisticsAlgorithm::initAlgorithm( const QVariantMap & )
+{
+  addParameter( new QgsProcessingParameterMultipleLayers( QStringLiteral( "INPUTS" ),
```

Comment:
```suggestion
  addParameter( new QgsProcessingParameterMultipleLayers( QStringLiteral( "INPUT" ),
```

Can you rename this parameter as "INPUT" to match the quasi-standard from other algorithms?